USE_CASE.md

```diff
@@ -0,0 +1,13 @@
+# AppDynamics Linux Monitoring Extension
+ 
+The Linux monitoring extension gathers metrics for a Linux machine and sends them to the AppDynamics Metric Browser. This extension works only with the standalone machine agent. It has been tested against Linux 2.6.32 on Ubuntu; info to be updated as tests against other distributions and Linux versions are completed.
+ 
+## Related Sandbox
+ 
+For reference only (Extension deployment is not possible on the current sandbox) [Cisco AppDynamics sandbox](https://devnetsandbox.cisco.com/RM/Diagram/Index/9e056219-ab84-4741-9485-de3d3446caf2?diagramType=Topology)
+ 
+## Links to DevNet Learning Labs
+ 
+[AppDynamics Fundamentals](https://developer.cisco.com/learning/modules/appdynamics-fundamentals)
+ 
+[Installation and Configuration workflow](https://github.com/Appdynamics/<extension-repo>/blob/master/README.md)
```

Comment:
Can you please fix the "Installation and Configuration workflow" and provide correct link to the repo's README.md